packages/utils/.eslintrc.json

```diff
@@ -18,7 +18,12 @@
       "files": ["*.json"],
       "parser": "jsonc-eslint-parser",
       "rules": {
-        "@nx/dependency-checks": ["error"]
+        "@nx/dependency-checks": [
+          "error",
+          {
+            "ignoredDependencies": ["benchmark"]
```

Comment:
Instead of doing this, we should exclude `perf/**/*` from production deps globally in [`nx.json`](https://github.com/flowup/quality-metrics-cli/blob/main/nx.json#L35-L42).

Reply:
added excludes for 

```sh
      "!{projectRoot}/perf/**/*",
      "!{projectRoot}/code-pushup.config.[(mj)jt]s"
```

Reply:
done